pkg/client/client.go

```diff
@@ -0,0 +1,29 @@
+package client
+
+import (
+	"github.com/bitnami-labs/charts-syncer/pkg/client/types"
+	"helm.sh/helm/v3/pkg/chart"
+)
+
+// Reader defines the methods that a ReadOnly chart client should implement.
+type Reader interface {
```

Comment:
This is a ChartReader right? it does not read bytes or other things, it reads charts.

Maybe we should not change it in this PR, but I wonder if we should change it eventually.

Reply:
It technically also reads relok8s bundles now but yes, that rename does not sound bad to me.